test/Microsoft.Health.Fhir.Tests.Integration/AssemblyInfo.cs

```diff
@@ -3,16 +3,6 @@
 // Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
 // -------------------------------------------------------------------------------------------------
 
-using Microsoft.Health.Fhir.Core.Features.Persistence.InMemory;
-using Microsoft.Health.Fhir.Tests.Common.Persistence;
+using Xunit;
 
-namespace Microsoft.Health.Fhir.Core.UnitTests.Persistence
-{
-    public class InMemoryStorageTests : FhirStorageTestsBase
-    {
-        public InMemoryStorageTests()
-            : base(new InMemoryFhirDataStore())
-        {
-        }
-    }
-}
+[assembly: TestFramework(typeName: "Microsoft.Health.Extensions.Xunit.FixtureArgumentSetsXunitTestFramework", assemblyName: "Microsoft.Health.Extensions.Xunit")]
```

Comment:
nit: Kinda weird that GitHub sees this as a rename/move?

Reply:
Not sure I can do anything about it...